bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -78,14 +79,22 @@
             description = "If true, the last modified value will be included in the sitemap.")
     private static final String PROP_INCLUDE_LAST_MODIFIED = "include.lastmod";
 
-    @Property(label = "Change Frequency Properties",
+    @Property(label = "Change Frequency Properties", unbounded = PropertyUnbounded.ARRAY,
```

Comment:
Change Frequency Properties and Priority Properties were existing in the current impl, and were being treated in code as multi-value, but the configs themselves were only simple strings.  As such, updated to make them configurable as multiple values.